commit 40f75db57467fb4b08b12ca2d43ebba4f26cee56 (refs/changes/43/3043/4)
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2017-12-11T16:31:16-08:00 (1 year, 10 months ago)
    
    WORKFLOW-221 upgrade to node v4
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/targets.json.in b/targets.json.in
index f2009ce..b68bfb2 100644
--- a/targets.json.in
+++ b/targets.json.in
@@ -517,9 +517,10 @@ cat <<EOF
     "image_name": "workflow",
     "image_description": "SDC Workflow",
     "image_version": "1.0.0",
-    "image_uuid": "fd2cc906-8938-11e3-beab-4359c665ac99",
+    "// image_uuid": "triton-origin-multiarch-15.4.1@1.0.1",
+    "image_uuid": "04a48d7d-6bb5-4e83-8c3b-e60a99e0f48f",
     "pkgsrc": [
-      "apg-2.2.3"
+      "apg-2.3.0bnb3"
     ],
     "repos": [
       {"url": "git@github.com:joyent/sdc-workflow.git"}
